test/Project.toml

```
@@ -12,3 +12,6 @@ RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
 SafeTestsets = "1bc83da4-3b8d-516f-aca4-4fe02f6d838f"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
+
+[compat]
+DDEProblemLibrary = "0.1.1"
```

Should be applied once the new version is available:
```suggestion
DDEProblemLibrary = "0.1.2"
```